ui/components/multichain/pages/permissions-page/permissions-page.js

```diff
@@ -77,13 +77,7 @@ export const PermissionsPage = () => {
           />
         }
       >
-        <Text
-          as="span"
-          variant={TextVariant.headingMd}
-          textAlign={TextAlign.Center}
-        >
-          {t('permissions')}
-        </Text>
```

Comment:
Removing extra html element to reduce html bloat and to maintain heading consistency